test_libs/pyspec/eth2spec/test/phase_0/block_processing/test_process_attestation.py

```diff
@@ -300,11 +304,12 @@ def test_non_empty_custody_bitfield(spec, state):
 
 @with_all_phases
 @spec_state_test
-def test_empty_aggregation_bitfield(spec, state):
+def test_empty_aggregation_bits(spec, state):
     attestation = get_valid_attestation(spec, state)
     state.slot += spec.MIN_ATTESTATION_INCLUSION_DELAY
 
-    attestation.aggregation_bitfield = b'\x00' * len(attestation.aggregation_bitfield)
+    attestation.aggregation_bits = Bitlist[spec.MAX_INDICES_PER_ATTESTATION](
```

Comment:
Doesn't the ssz type handle the casting here?
```suggestion
    attestation.aggregation_bits = [0b0] * len(attestation.aggregation_bits)
```

Reply:
I don't think that python does any typecasting on assignment, that would be very un-pythonic (they are only type annotations, after all). The only exceptions is slice assignments (which aren't really assignments)

Reply:
you're right. don't know what I was thinking!

Reply:
It's unpythonic. But support is there to do implicit type casting for integers, and other types, where we really wouldn't want to for readability.